### DISCUSSION TITLE
Welcome to Datafeedr Discussions!

### DISCUSSION THREAD

**Discussion by EricBusch** [datafeedr/wordpress-plugins#1] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about yourself.

**Comment by PriemK** [comment]
Hi guys, 
Kevin here, happily working with Datafeedr for some time now on different websites. Really hope you keep developing the software.
Here to post some requested features and (expert?) questions/issues with Datafeedr
